src/react-apps/applications/shared/src/file-editor/MonacoEditorComponent.tsx

```diff
@@ -118,31 +145,39 @@ class MonacoEditorComponent extends React.Component<IMonacoEditorComponentProps,
         )
         :
         (
-          <MonacoEditor
-            theme={'editorTheme'}
-            height={this.props.heightPx ? this.props.heightPx : '100%'}
-            width={this.props.widthPx ? this.props.widthPx : '100%'}
-            value={this.props.value}
-            language={this.props.language}
-            options={
-              {
-                autoClosingBrackets: 'always',
-                autoIndent: true,
-                automaticLayout: true,
-                colorDecorators: true,
-                minimap: {
-                  enabled: false,
-                },
-                cursorBlinking: 'smooth',
-                scrollbar: {
-                  vertical: 'auto',
-                },
-                scrollBeyondLastLine: false,
+          <div
+            ref={this.state.monacoWrapperRef}
+            tabIndex={0}
+            onFocus={this.setFileEditorFocus('focus')}
+            onBlur={this.setFileEditorFocus('blur')}
+            onKeyDown={this.handleKeyPress}
+          >
+            <MonacoEditor
+              theme={'editorTheme'}
+              height={this.props.heightPx ? this.props.heightPx : '100%'}
+              width={this.props.widthPx ? this.props.widthPx : '100%'}
+              value={this.props.value}
+              language={this.props.language}
+              options={
+                {
+                  autoClosingBrackets: 'always',
+                  autoIndent: true,
+                  automaticLayout: true,
+                  colorDecorators: true,
+                  minimap: {
+                    enabled: false,
+                  },
+                  cursorBlinking: 'smooth',
+                  scrollbar: {
+                    vertical: 'auto',
+                  },
+                  scrollBeyondLastLine: false,
+                }
               }
-            }
-            onChange={this.props.onValueChange}
-            editorWillMount={this.editorWillMount}
-          />
+              onChange={this.props.onValueChange}
```

Comment:
Some weird spacing? Should be aligned with rest of props?

Reply:
No. Line 177 is aligned with the "{" on line 176.

Reply:
@lorang92 Not using the VSCode Plugin? ;)